src/problems/basic_problems.jl

```diff
@@ -378,24 +378,23 @@ struct QuadratureProblem end
 @deprecate QuadratureProblem(args...; kwargs...) IntegralProblem(args...; kwargs...)
 
 @doc doc"""
-
 Defines an optimization problem.
 Documentation Page: https://docs.sciml.ai/Optimization/stable/API/optimization_problem/
 
 ## Mathematical Specification of an Optimization Problem
 
-To define an Optimization Problem, you simply need to give the function ``f``
-which defines the cost function to minimize:
+To define an optimization problem, you need the objective function ``f``
+which is minimized over the domain of ``u``, the collection of optimization variables:
 
 ```math
 min_u f(u,p)
 ```
 
-``u₀`` is an initial guess of the minimum. `f` should be specified as `f(u,p)`
-and `u₀` should be an AbstractArray (or number) whose geometry matches the
-desired geometry of `u`. Note that we are not limited to numbers or vectors
+``u₀`` is an initial guess for the minimizer. `f` should be specified as `f(u,p)`
+and `u₀` should be an `AbstractArray` whose geometry matches the
+desired geometry of `u`. Note that we are not limited to vectors
 for `u₀`; one is allowed to provide `u₀` as arbitrary matrices /
-higher-dimension tensors as well.
+higher-dimension tensors as well if the optimization solver supports it.
```

Comment:
which optimizers don't support it? why is it not vec/reshaped?

Reply:
Just not implemented